src/main/java/smokefree/domain/Initiative.java

```diff
@@ -253,6 +255,21 @@ private void validatePlaygroundsRange(GeoLocation newPlaygroundLocation, long di
                             "playground does already exists within "+ SmokefreeConstants.MAXIMUM_PLAYGROUNDS_DISTANCE+ " Meters");
                 });
     }
+    private void validateMaximumAllowedVolunteers() {
+        if(citizens.stream().count() >= SmokefreeConstants.PlaygroundWorkspace.MAXIMUM_VOLUNTEERS_ALLOWED) {
```

Comment:
why not just do .size() instead of .stream().count() ?

Reply:
made changes